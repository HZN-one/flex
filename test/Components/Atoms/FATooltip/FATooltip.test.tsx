/**
 * @jest-environment jsdom
 */

import * as React from "react";
import "@testing-library/jest-dom";

import { FATooltip } from "@Components";

import { fireEvent, render } from "@testing-library/react";

describe("FATooltip", () => {
  it("should match snapshot", () => {
    const { container } = render(
      <FATooltip title="test" testID="tooltip-example">
        <button>test</button>
      </FATooltip>
    );
    expect(container).toMatchSnapshot();
  });
});
