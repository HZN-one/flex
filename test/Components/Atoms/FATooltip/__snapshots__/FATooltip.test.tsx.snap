// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`FATooltip should match snapshot 1`] = `
<div>
  <button
    aria-label="test"
    class=""
    data-mui-internal-clone-element="true"
    data-testid="tooltip-example"
  >
    test
  </button>
</div>
`;
